tests/acceptance/features/apiGraph/addUserToGroup.feature

```diff
@@ -149,3 +149,44 @@ Feature: add users to group
   Scenario: admin tries to add user to a group without sending the group
     When the administrator tries to add user "Alice" to group "" using the Graph API
     Then the HTTP status code should be "404"
+
+
+  Scenario: add multiple users to a group at once
+    Given the administrator has given "Alice" the role "Admin" using the settings api
+    And these users have been created with default attributes and without skeleton files:
+      | username |
+      | Brian    |
+      | Carol    |
+    And user "Alice" creates a group "grp1" using the Graph API
+    When the administrator "Alice" adds the following users to a group "grp1" at once using the Graph API
+      | username |
+      | Brian    |
+      | Carol    |
+    Then the HTTP status code should be "204"
+    And the following users should be listed in the following groups
+      | username | groupname |
+      | Brian    | grp1      |
+      | Carol    | grp1      |
+
+
+  Scenario: admin tries to add users at once to a group without sending the group
+    Given the administrator has given "Alice" the role "Admin" using the settings api
+    And these users have been created with default attributes and without skeleton files:
+      | username |
+      | Brian    |
+      | Carol    |
+    When the administrator "Alice" adds the following users to a group "" at once using the Graph API
```

Comment:
IMO better to create separate steps for negative cases. We can add examples for a non-existent group also.

Reply:
similar other steps are handled in the same way so its a topic of discussion. @saw-jan @SagarGi @saw-jan . If we make separate step then we have to refactor other step too which has been added in old PR

Reply:
This is a newly added step. So, it would be far better if we can use separate step for this e.g.:
`When the administrator "Alice" tries to add the following users to a non-existing group at once using the Graph API`
This way, we don't need to try-catch or use conditions with a single-step def function. Also, it isolates the positive and negative case steps.

We can refactor the already-added ones later.

Reply:
I had create an issue to work on the refactor https://github.com/owncloud/ocis/issues/5646